packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VisXZoom/VisXZoom.js

```diff
@@ -228,31 +221,29 @@ function VisXZoom({
       {_zoom => {
         zoomRef.current = _zoom
         return (
-          <g ref={wheelEventLayer}>
-            <ZoomingComponent
-              initialTransformMatrix={_zoom.initialTransformMatrix}
-              transformMatrix={_zoom.transformMatrix}
-              transform={_zoom.toString()}
-              move={move}
-              {...props}
-            >
-              <ZoomEventLayer
-                focusable
-                height={height}
-                onDoubleClick={onDoubleClick}
-                onKeyDown={onKeyZoom}
-                onPointerEnter={onPointerEnter}
-                onPointerDown={panning ? _zoom.dragStart : DEFAULT_HANDLER}
-                onPointerMove={panning ? _zoom.dragMove : DEFAULT_HANDLER}
-                onPointerUp={panning ? _zoom.dragEnd : DEFAULT_HANDLER}
-                onPointerLeave={onPointerLeave}
-                onWheel={onWheel}
-                panning={panning}
-                tabIndex={0}
-                width={width}
-              />
-            </ZoomingComponent>
-          </g>
+          <ZoomEventLayer
+            disabled={!zooming || !move}
+            focusable
+            height={height}
+            onDoubleClick={onDoubleClick}
+            onKeyDown={onKeyDown || onKeyZoom}
+            onPointerEnter={onPointerEnter}
+            onPointerDown={panning ? _zoom.dragStart : DEFAULT_HANDLER}
+            onPointerMove={panning ? _zoom.dragMove : DEFAULT_HANDLER}
+            onPointerUp={panning ? _zoom.dragEnd : DEFAULT_HANDLER}
+            onPointerLeave={onPointerLeave}
+            onWheel={onWheel}
+            panning={panning}
+            tabIndex={0}
+            width={width}
+          >
+            {children({
+              initialTransformMatrix: _zoom.initialTransformMatrix,
+              transformMatrix: _zoom.transformMatrix,
+              transform: _zoom.toString(),
+              move
+            })}
```

Comment:
The React docs recommend using a render function to make it explicit that a component injects props into its children.
https://react.dev/reference/react/cloneElement#alternatives

It means you have to write wrapped children as a function:
```jsx
<VisXZoom>
  {(zoomProps) => <SVGImageCanvas {...zoomProps} />}
</VisXZoom>
```